src/uct/ib/rc/accel/rc_mlx5_common.h

```diff
@@ -275,7 +275,8 @@ uct_rc_mlx5_txqp_inline_post(uct_rc_iface_t *iface, enum ibv_qp_type qp_type,
                              unsigned opcode, const void *buffer, unsigned length,
                   /* SEND */ uint8_t am_id, uint64_t am_hdr, uint32_t imm_val_be,
                   /* RDMA */ uint64_t rdma_raddr, uct_rkey_t rdma_rkey,
-                  /* AV   */ uct_ib_mlx5_base_av_t *av, size_t av_size
+                  /* AV   */ uct_ib_mlx5_base_av_t *av, size_t av_size,
```

Comment:
i would rename/order this as follows for better consistency:
```
uct_ib_mlx5_base_av_t *base_av,  
struct mlx5_grh_av *grh_av,
size_t av_size
```

Reply:
done